### ISSUE TITLE
Create a set of test images and investigate a diffing platform to ensure rendering parity

### ISSUE THREAD

**Issue by DHowett-MSFT** [Microsoft/WinObjC#1271]
1. Write rendering code to run on WinObjC and the reference platform.
2. Render to a bitmap.
3. Visually compare the bitmaps to establish a baseline.
4. Use the new baselined images from WinObjC as input to an image diff engine against future renders.

**Comment by DHowett-MSFT** [comment]
Blocked by #1124

**Comment by DHowett-MSFT** [comment]
Right now, greenlines are being generated manually:

```ps1
gci TestIm* | %{
    magick compare -highlight-color green -lowlight-color white "reference\$($_.Name)" $_.Name -compose src Compared_$($_.Name)
    magick montage -label "Reference" "reference\$($_.Name)" -label "WinObjC" $_.Name -label "Diffs" Compared_$($_.Name) -tile 3x1 -geometry +0+0 "FINAL_$($_.Name)"
}
```

Future work: Generate greenlines automatically.

**Comment by DHowett-MSFT** [comment]
This completed as of b67d620.
